src/translations.js

```diff
@@ -37,6 +37,25 @@ const translations = {
       speaker_management: 'Gestion des groupes',
     },
   },
+  hu: {
+    placeholder: {
+      tts: 'Szövegfelolvasás',
+    },
+    label: {
+      leave: 'Kilépés',
+      ungroup: 'Összes kilép',
+      group_all: 'Összes belép',
+      send: 'Küldés',
+      master: 'forrás',
```

Comment:
Please capitalize this, that way we can easily reuse the same string in the future, without the need for extra styling rules.

Reply:
I was about to notice that it shouldn't be capitalised in English either, but I accept your approach.